internal/cli/command_factory.go

```diff
@@ -76,11 +76,15 @@ func (factory *CommandFactory) Build(command CommandDefinition) *cobra.Command {
 				os.Exit(1)
 			}
 
-			factory.telemetryService = telemetry.NewService(
+			telemetryService := telemetry.NewService(
```

Comment:
I'm guessing you originally changed this b/c you introduced an error return value to `NewService`, but since thats not needed can we revert this back to how it was?